tests/acceptance/features/apiLocks/unlockFiles.feature

```diff
@@ -63,7 +63,6 @@ Feature: unlock locked items
       | new              |
       | spaces           |
 
-  @issue-7638
```

Comment:
@2403905 Let's not remove the issue tags. it servers as a reference for the future.